tools/perfscale-audit/metric-client/metric-client.go

```diff
@@ -36,8 +36,20 @@ import (
 )
 
 const (
-	vmiCreationTimePercentileQuery   = `histogram_quantile(0.%d, rate(kubevirt_vmi_phase_transition_time_from_creation_seconds_bucket{phase="Running"}[%ds]))`
-	resourceRequestCountsByOperation = `increase(rest_client_requests_total{pod=~"virt-controller.*|virt-handler.*|virt-operator.*|virt-api.*"}[%ds])`
+	vmiCreationTimePercentileQuery = `histogram_quantile(0.%d, rate(kubevirt_vmi_phase_transition_time_from_creation_seconds_bucket{phase="Running"}[%ds]))`
+)
+
+// Counter - When using a Counter, use a range vector type so that we only get data
+//           from the current test period instead of the cumulative count.  Using an
+//           `offset` will covert to a range vector.
+const (
+	resourceRequestCountsByOperation = `sum by (verb, resource) (rest_client_requests_total{pod=~"virt-controller.*|virt-handler.*|virt-operator.*|virt-api.*"} offset %ds)`
```

Comment:
I don't think `offset` is doing what we want here.  Offset allows us to execute a command in the past. If we call offset with a time interval that places the execution right before the density test occurs, then we're getting only the sum of values at that specific point in time in the past.

Reply:
ya .. I think the example I looked at misled me.  I guess `increase` is the only option here I can currently think of.  I'll change back to `increase` for now and maybe I'll come up with in alternative later